src/app/home-page/top-level-community-list/top-level-community-list.component.ts

```diff
@@ -12,6 +12,7 @@ import { PaginationComponentOptions } from '../../shared/pagination/pagination-c
 import { hasValue } from '../../shared/empty.util';
 import { switchMap } from 'rxjs/operators';
 import { PaginationService } from '../../core/pagination/pagination.service';
+import { environment } from 'src/environments/environment';
```

Comment:
Same comment as above: please inject the environment instead of directly importing it